app/src/main/java/acr/browser/lightning/browser/activity/BrowserActivity.kt

```diff
@@ -3850,6 +3850,7 @@ abstract class BrowserActivity : ThemedBrowserActivity(), BrowserView, UIControl
 
             }
         }
+        if (tabsManager.currentTab == null) return false
```

Comment:
I'm not too happy with that fix. Can you post the actual crash logs on the bug report? Do you have a call stack?
At the very least we would need a code comment with a link to the issue.

Reply:
I also don't like it, hence my comment in the issue: "I'm not sure whether this is a good solution, as it seems more like a workaround"

Reply:
I'm with you man. Sometimes workaround is all we have or all we can afford because there is only so much time we can invest in such issues 😁
However in this case I had to investigate as it touches an area I already had to do some funny things with. Namely the swipe refresh layout target. See my posts on #285.